SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -4668,6 +4681,13 @@ void CEulerSolver::BC_Riemann(CGeometry *geometry, CSolver **solver_container,
 
       GetFluidModel()->SetTDState_rhoe(Density_i, StaticEnergy_i);
 
+      /*--- Set initial values for density and energy for Newton solvers in fluid model ---*/
+      if(config->GetKind_FluidModel() == DATADRIVEN_FLUID){
+        su2double relax_Newton = config->GetRelaxation_DataDriven();
+        GetFluidModel()->SetInitialDensity(relax_Newton*Density_i + (1 - relax_Newton)*config->GetDensity_Init_DataDriven());
+        GetFluidModel()->SetInitialEnergy(relax_Newton*StaticEnergy_i + (1 - relax_Newton)*config->GetEnergy_Init_DataDriven());
+      }
+      
```

Comment:
Why is this used for boundary conditions but not when computing primitives for the entire domain?

Reply:
Unresolving again.

Reply:
@EvertBunschoten if this makes it to 2 months without a reply I'll start reverting what was merged into TestCases already.